src/ui/utils/timeline.test.ts

```diff
@@ -231,3 +232,32 @@ describe("filterToFocusRegion", () => {
     ]);
   });
 });
+
+describe("mergeSortedPointLists", () => {
+  fit("will correctly interleave overlapping ranges", () => {
```

Comment:
We should probs add a line rule to prevent `fit` or `test.only` from being committed accidentally. I do this sometimes too.